### PR TITLE
SqlTask result email should not throw exception

### PR DESCRIPTION
**Pull request by lonkin1**
## What is the purpose of the pull request

This pull request adds ShowType NONE
This pull request update SqlTask ,use log log instead of exception  when send email

## Verify this pull request

This pull request is already covered by existing tests, such as SqlExecutorTest.

### CODE REVIEW

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/ShowType.java [hunk 1]
```diff
@@ -23,11 +23,13 @@
  */
 public enum ShowType {
     /**
+     * -1 NONE;
      * 0 TABLE;
      * 1 TEXT;
      * 2 attachment;
      * 3 TABLE+attachment;
      */
+    NONE(-1, "none"),
```
Comment (CalvinKirs) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/ShowType.java:32]: In my opinion, it is not needed.

**Comment by qiaozhanwei** [comment]
At the beginning of the design, it is necessary to send emails for SQL query tasks, otherwise the SQL will be meaningless. If the email fails to be sent, the task will fail. Have other advice ?

设计之初是这样的，对于SQL的查询任务是需要发送邮件的，否则SQL将无意义，如果邮件发送失败，任务将失败。你有更好的建议么？

**Comment by gabrywu** [comment]
> At the beginning of the design, it is necessary to send emails for SQL query tasks, otherwise the SQL will be meaningless. If the email fails to be sent, the task will fail. Have other advice ?
> 
> 设计之初是这样的，对于SQL的查询任务是需要发送邮件的，否则SQL将无意义，如果邮件发送失败，任务将失败。你有更好的建议么？

Agree with @qiaozhanwei , please create an issue to discuss about it. not suggest to merge this PR

**Comment by gabrywu** [comment]
@lonkin1  I closed this PR firstly until you give more details and more solutions, feel free to reopen this PR if necessary

**Comment by lonkin1** [comment]
When the SQL operation is successful, the processing of the results should not block the whole process. It is recommended to add policies to avoid throwing exceptions directly.

---Original---

**Comment by gabrywu** [comment]
@lonkin1 you can split the SQL task from original DAG to avoid throwing exceptions

**Pull request closed without merging**